.pre-commit-config.yaml

```diff
@@ -1,24 +1,46 @@
 repos:
--   repo: https://github.com/psf/black
+  - repo: https://github.com/psf/black
     rev: 22.3.0
     hooks:
-    - id: black
-      args: ["cfgov", "--line-length=79"]
-      exclude: migrations
--   repo: https://gitlab.com/pycqa/flake8
+      - id: black
+        args: ['cfgov', '--line-length=79']
+        exclude: migrations
+  - repo: https://gitlab.com/pycqa/flake8
     rev: 4.0.1
     hooks:
-    - id: flake8
-      additional_dependencies: [flake8-bugbear==22.4.25]
--   repo: https://github.com/pycqa/isort
+      - id: flake8
+        additional_dependencies: [flake8-bugbear==22.4.25]
+  - repo: https://github.com/pycqa/isort
     rev: 5.10.1
     hooks:
       - id: isort
         name: isort (python)
-        args: ["cfgov"]
--   repo: https://github.com/PyCQA/bandit
+        args: ['cfgov']
+  - repo: https://github.com/PyCQA/bandit
     rev: 1.7.4
     hooks:
-    - id: bandit
-      args: ["-c", "pyproject.toml", "--recursive"]
-      additional_dependencies: [toml==0.10.2]
+      - id: bandit
+        args: ['-c', 'pyproject.toml', '--recursive']
+        additional_dependencies: [toml==0.10.2]
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.7.1
+    hooks:
+      - id: prettier
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: v8.28.0
+    hooks:
+      - id: eslint
+        additional_dependencies:
+          - eslint@8.26.0
```

Comment:
Is it correct that the `v8.28.0` and `eslint@8.26.0` are out of sync? Looks like 8.28.0 of eslint is out https://github.com/eslint/eslint/releases/tag/v8.28.0

Reply:
yeah, I had just copied the eslint stuff from our package.json, which was still on 8.26.0